src/components/footer/styledFooter.js

```diff
@@ -65,6 +100,4 @@ export const Link = styled.a`
 `
 export const FooterBottom = styled.div`
```

Comment:
the 'terms of use' text should be under the 'awesome container company' text at 375. I would add flex-direction to solve this.